src/ray/common/asio/instrumented_io_context.h

```diff
@@ -61,6 +61,23 @@ struct GuardedGlobalStats {
   mutable absl::Mutex mutex;
 };
 
+/// An opaque stats handle, used to manually instrument event loop handlers that don't
+/// hook into a .post() call.
+struct StatsHandle {
+  std::string handler_name;
+  int64_t start_time;
+  std::shared_ptr<GuardedHandlerStats> handler_stats;
+  std::shared_ptr<GuardedGlobalStats> global_stats;
+
+  StatsHandle(std::string handler_name_, int64_t start_time_,
```

Comment:
Why do we use `_` for the argument and not append `_` for private attribute? Isn't our convention opposite?

Reply:
Because the attributes are public, and this constructor is an internal hack to make `std::make_shared<StatsHandle>(handler_name, start_time, handler_stats, global_stats)` work prior to C++20 (where it works without needing this constructor), so I thought I'd use the `_` suffix for the constructor arguments in order to keep the public members clean. Lmk if you have a better idea for this other than adding the `_` suffix to the public members, which I don't think we should do since they're public.